recipe/build.sh

```diff
@@ -6,59 +6,31 @@ set -ex
 # https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
 export CXXFLAGS="$(echo "${CXXFLAGS}" | sed -E 's@-std=c\+\+[^ ]+@@g') -D_LIBCPP_DISABLE_AVAILABILITY"
 
-if [[ "$SHLIB_EXT" == '.dylib' ]]; then
-
-  if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
   BUILD_TEST=ON
-  else
+else
   BUILD_TEST=OFF
-  fi
-
-  export CMAKE_BUILD_TYPE=Release
-  export BUILD_SHARED_LIBS=ON
-  export CMAKE_BUILD_WITH_INSTALL_RPATH=ON
-  export CMAKE_CXX_STANDARD=17
-  export TMPDIR="${SRC_DIR}\build\tmp"
-
-  mkdir build
-  pushd build
-  mkdir tmp
-
-  cmake -GNinja ${CMAKE_ARGS} \
-      -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
-      -DCMAKE_PREFIX_PATH="${PREFIX}" \
-      -DOPENFST_BUILD_TEST=$BUILD_TEST \
-      ..
-
-
-  cmake --build . --verbose --config Release -- -v -j ${CPU_COUNT}
-  ctest --rerun-failed --output-on-failure
-  cmake --install . --verbose --config Release
+fi
 
+export CMAKE_BUILD_TYPE=Release
+export BUILD_SHARED_LIBS=ON
+export CMAKE_BUILD_WITH_INSTALL_RPATH=ON
+export CMAKE_CXX_STANDARD=17
+export TMPDIR="${SRC_DIR}\build\tmp"
 
-  popd
-else
-  # Get an updated config.sub and config.guess
-  cp $BUILD_PREFIX/share/gnuconfig/config.* .
-  if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
-    OPENFST_CROSS_COMPILATION_CONFIGURE_OPTS="--build=${BUILD} --host=${HOST}"
-  else
-    OPENFST_CROSS_COMPILATION_CONFIGURE_OPTS=""
-  fi
+mkdir build
+pushd build
+mkdir tmp
 
-  ./configure \
```

Comment:
FWIW, if we're switching from autotools to cmake, we have a [checklist](https://conda-forge.org/docs/maintainer/knowledge_base/#moving-from-an-autotools-build-to-a-cmake-build) that should be done, to avoid creating any ABI breaks. You'll probably want to switch on artefact persistence so that you can download a successful build from the PR and double-check.

Reply:
What if we kept autotools for linux-64 but moved to cmake for linux-aarch64 and linux-ppc64le?

Reply:
You cannot break what didn't exist before, so then there's no worry. I don't mind changing to CMake (though not a maintainer here, just a personal opinion), but if we do that we should check that things keep working as before.

Reply:
Ok - that's what I've done. Probably easier than going through the whole checklist, and I've messed that one up before ..